src/App.js

```diff
@@ -1,15 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
-function App() {
+
+const App = () => {
+  const [todoList, setTodoList] = useState([]);
+  const addTodo = (newTodo) => {
```

Comment:
If you run the app in the browser and console log the newTodo argument, you'll see that there's no id property in the object - do you mind fixing that?